Resources/Prototypes/Maps/rim.yml

```diff
@@ -38,7 +38,7 @@
             Chemist: [ 3, 3 ]
             MedicalDoctor: [ 4, 4 ]
             MedicalIntern: [ 4, 4 ]
-            Paramedic: [ 3, 3 ]
+            Paramedic: [ 2, 2 ]
```

Comment:
still too much, we have 2 parameds like only for Kettle, which is huge map

Reply:
My map has a room for paramedics a "fire department." This is a intentional feature of my map